scripts/check-fuzzer.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env sh
 
 main() {
+    echo " >> Getting random bytes from /dev/urandom"
+    mkdir -p utils/runtime-fuzzer/fuzz/corpus/main
+    dd if=/dev/urandom of=utils/runtime-fuzzer/fuzz/corpus/main/check-fuzzer-bytes bs=1 count=27000000
```

Comment:
Please describe here why we should fill this file with random bytes

Reply:
done